easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+# Updated: Denis Kristak (Inuits)
+# Update to a Python3 version
+easyblock = 'PackedBinary'
+
+name = 'CPC2'
+version = '0.1'
+local_commit = '813487e'
+
+homepage = 'http://cpc2.cbi.pku.edu.cn'
+description = """ a fast and accurate coding potential calculator based on sequence intrinsic features  """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# original sources at http://cpc2.cbi.pku.edu.cn/download.php are currently not available
+# download from github mirror
+source_urls = ['https://github.com/lucventurini/CPC2/archive/']
```

Comment:
This seems to be available? https://github.com/gao-lab/CPC2_standalone/releases
I was directed here by going to the old website, which redirected to the new gao-lab website, which redirected here. 
This seems much much newer as well compared to the version 0.1 in the lucventurini repo, and this one has been ported to python 3 already.

Reply:
@deniskristak @smoors Thoughts on going with a more recent version available via https://github.com/gao-lab/CPC2_standalone/releases ?

Reply:
@boegel just pushed an updated version using the newer source. I had to change version too, so I'm removing the original files.